sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -934,6 +934,15 @@ object SQLConf {
       .intConf
       .createWithDefault(10000)
 
+  val GENERATED_CLASS_LENGTH_THRESHOLD =
+    buildConf("spark.sql.codegen.generatedClass.size.threshold")
+      .doc("Threshold in bytes for the size of a generated class. If the generated class " +
```

Comment:
Please add `.internal()`. I think this is not a parameter for users.

Reply:
if users cannot set it, how can this parameter be changed?

Reply:
I think this is not a frequently-used parameter for users. Also, other JVM implimentation-specific parameters are internal, e.g., `WHOLESTAGE_HUGE_METHOD_LIMIT`. cc: @rednaxelafx

Reply:
I agree with you, but might you please explain me how to set a parameter which is internal? Thanks.

Reply:
Users can change this value in the same way with other public parameters, but  `internal` just means that parameters are not explicitly exposed to users.

Reply:
thanks for the explanation, I'll add it immediately.